libzeth/test/powersoftau_test.cpp

```diff
@@ -25,12 +25,12 @@ fs::path g_testdata_dir = fs::path("..") / "testdata";
 
 static std::string hex_to_bin(const std::string &hex)
 {
-    return hexadecimal_str_to_binary_str(hex);
+    return hex_to_bytes(hex);
 }
 
 static std::string bin_to_hex(const std::string &hex)
 {
-    return binary_str_to_hexadecimal_str(hex);
+    return bytes_to_hex(hex);
```

Comment:
This is a bit misleading. The name of these functions let the reader assume that they manipulate binary strings, while they seem to expect strings of bytes. Moreover shouldn't we have:
`static std::string bin_to_hex(const std::string &bin)` instead of `static std::string bin_to_hex(const std::string &hex)`?

Reply:
These have been removed now.